src/NodeServices/GraphicsDataInterfaces.cs

```diff
@@ -48,7 +48,15 @@ public interface IRenderPackage
         /// The number of mesh vertices in the package.
         /// </summary>
         int MeshVertexCount { get; }
-        
+
+        //TODO(mike) uncomment this code when we decide to add transforms to the public API
```

Comment:
Alternatively, you can define a new interface ITransformableRenderPackage derived from IRenderPackage or just ITransformable without deriving from any other interface and cast the incoming render package object to this new interface for use.